src/Symfony/Bundle/FrameworkBundle/Resources/config/debug.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\NotTaggedControllerValueResolver;
+use Symfony\Component\HttpKernel\Controller\TraceableArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\TraceableControllerResolver;
+use Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('debug.event_dispatcher', TraceableEventDispatcher::class)
+        ->decorate('event_dispatcher')
```

Comment:
We need an extra indent after each `->set()`, check CS style in this PR: https://github.com/symfony/symfony/pull/36778/files